const getUserController = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
const getUserSingleController = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
const postUserCreateController = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
const putUserUpdateController = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
const patchUserUpdateController = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
const deleteUserDeleteController = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
const userChangePasswordController = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};

module.exports = {
  getUserController,
  getUserSingleController,
  postUserCreateController,
  putUserUpdateController,
  patchUserUpdateController,
  deleteUserDeleteController,
  userChangePasswordController,
};
